### ISSUE TITLE
Missing dependency in 4.2.9

### ISSUE THREAD

**Issue by huard** [geopython/pywps#564]
# Description

sphinx is not in requirements but is imported by pywps.test.

This does not seem to be a problem in `master`, but not sure if it because the issue was fixed, or the commit not pulled into master. 

# Steps to Reproduce
```bash
conda create -n pywps4.2.9 python=3.7
conda activate pywps4.2.9
python setup.py develop
python
```
Then 
```python
from pywps import tests
Python 3.7.9 | packaged by conda-forge | (default, Dec  9 2020, 21:08:20) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pywps import tests
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/david/src/pywps/pywps/tests.py", line 15, in <module>
    from pywps.ext_autodoc import MetadataUrl
  File "/home/david/src/pywps/pywps/ext_autodoc.py", line 2, in <module>
    from sphinx.ext.autodoc import ClassDocumenter, bool_option
ModuleNotFoundError: No module named 'sphinx'
```
```





# Additional Information

**Comment by tlvu** [comment]
Sphinx is in https://github.com/geopython/pywps/blob/20e1e254a3f7914e555fa89f363d1f6eb5f3895c/requirements-dev.txt#L4 so that might explain why it's passing on Travis-CI https://travis-ci.org/github/geopython/pywps/builds/750280910

**Comment by tlvu** [comment]
That's coming from my commit https://github.com/geopython/pywps/commit/2a55b6e95f51c648dc94bf3c89db7370b56c1c9c, so should I have created the `MetadataUrl` class in `pywps.app.Common` instead?

**Comment by tlvu** [comment]
I introduced the commit above in this PR https://github.com/geopython/pywps/pull/542.  It was my first time touching pywps code. The class `MetadataUrl` should probably be in `pywps.app.Common` together with `Metadata` instead.  Will submit a PR for this.

**Comment by cehbrecht** [comment]
fixed by PR #565
